### ISSUE TITLE
Henrystats - Identifying Safe Airdrop Farmers (272 Addresses)

### ISSUE THREAD

**Issue by henrystats** [safe-global/safe-user-allocation-reports#371]
# Related Safe Addresses

```
0x6de75bb282898ff5e28e308ee7f8750a0f2b077e
0x17b2717ecc7fbc1538cd268cca7b395541f4a2c7
0x14ab82062dc2622c4f1467042652fb7110b92d53
0xb30b700332b27438c78e95d3ac714542ca6c4cb1
0xfb5637d9aa502c0ac6779e87428ad735750d1f3d
0x0ac7d3f641ff2d6fb5445c2d268ec16be3e1851a
0xfd30d37f7f70faa2025a3931a250f39309d07f15
0xefcf76d61503892e184665db37746054670f9966
0x6847b3a7d1d3915dfe361cbb7fb01d79253adde9
0xa8a6959e25044fd0c8334c98c6e5c7d46a21b950
0xc8c5899103a0803a95da06e4ab44219d3b56a834
0xb580483068d3a808190287466ab0e524efb908d6
0x3ddf8ecb1e65575bcd2439fb085c5c61cff16215
0x6c51954dc6c652ba46f833f1cc66ec4d1ad2d3a2
0x27b838c2e151e45b82828630841dc58c75929540
0xa73cf67a9970d1e42d66ef1c547e92c03a261537
0xefbcf693798bc9c4c0bc74a1410ff805f95e8332
0x1365ab2ec4eaf0c0fb0e78f08074d4b5fdefcadd
0x03882a98090de9f6337a149f62d1627eb8321bd8
0xfcd869624a52f23e45cf433d418a5d8b604ca185
0xdb7a2deef12a4fb799a3688dc942d92c2feb49aa
0xb847984bec8ffe7851af125042f8a56b682e960c
0xec7bb9cba553cd672b57174e2eb30730d1100f31
0x5052336f3627883234e3e2bf60308ce83880fde8
0x9ca0c1d5740eab234e8040f181000b1cee065f4a
0x8fccfec5141b8703acacfdf97845546c3cda48c5
0x44f50a1212fea781c0a382390a6bc91d33a66ed3
0x9ee36996a48b07d33dad33e8bf3b737510dbd3f4
0x2502059ec076c5409d7332f57d6f1386710efcb4
0x67d91100fc1ff6b640c8148c302ab9864e34938b
0x6d79c1c14370a9aa3e84dfd97efe257aa18b00b9
0x78190217972949feb28cd80c2848db735ff31b3d
0x418f18b6ec449296322cba386fe0e94ffca10d32
0xb0c67d860158e0592b317488685fbcb3f836c8d6
0x21c3675c075cb516c52fb2da9c78d0fe0b4b72f8
0xc8cd18825ca8569f60284a469549f958584e7fc7
0x2dd08fae19b8ac43555dab8d46cd77311e3e0a03
0x993d5adcb4bf8e848ba25a42c92934f59cff5f0a
0xc44b3e6889c08d0df9ef494ad8a7cbe176896d11
0xbef0e77e3c2c3d07c02ba8a419dab3d95c74dec9
0x48ccbfd8bea26e852c6db1017b6570dd996e2147
0x32e7aa624c247ab1495d99b1f7d87490f85294ca
0x5ac3b6d06495d48d8d8db55a85aeff39c8d974c7
0x9e544eeffd40bfc395e88a59de0344bdc21e7d36
0x9eab16cb42c9d118f98b849fee341bb80f0d3f28
0xb8ab0e9a4a5ab79ca7820ddfae6e46107904319a
0xb7f3d6672326711e39f6431878c4a9704a1502a0
0x2c9fc543242ccf46487204fcbca904353ecdd10d
0x5b05e0317fe05d16d9fb819c712ceb23d3894bba
0xcc191076ef2337ccfa15ce7986e68d45a4144ee8
0x4469e2bcc3578a66e2593fdc63845d2dd57f8ba7
0xd004e03bb12b4422bc3e02ce315c65dc9d82cd49
0xfa422bab5bd8e9ba672e3b1148a04db69c92f2fc
0xb54af45f64c2308046fd76346dd48970569e06d0
0x4651b4a07f645a1030a73675071697033fc4cbee
0x24f381e982333515200f25e586ac6410dab002c5
0x36a10c0e8278d549603cba4ebbf9ab290df93415
0x4d6c1ee2e18abe9fff9c77b3ad65d172d68522c9
0x4605db3c81fbae12905eba1196da819b56f24f16
0x53cc9dd3ded8cf62dbc7946d25b901b7adcdcf78
0xa71ba4280474e6ee432ec84ab27d727e8c2f9c27
0xbb2a181469b94450b51409178d9c8dddcb5344de
0x9f894f5e4f2d8e5b953fb6c01691f25ecd9e1ec0
0xe738628737657ddefb4aef226e6190e90ed564b6
0xf0b2802d6010f53a0ba7fdee1c0f036c91a2d5b5
0x724e08c98256a570aadbac6d20c1b9ccce208f28
0x55b7c326ca5fd26f96b485df37da9eb911294652
0xb0b8cfdb5b4a56d0c12995652be8d4de93478dbc
0x552d29e348bd8de64c260081c16a1bc2aa992666
0x262623d6e46894fbebf1f53d832788a53d2e490f
0x6cfaecf069bcaea72daa8ad9965335d9d6c823b4
0x04e35228541d895cdd6383e9ce3bd2f5dc2b028d
0xb65913cafc6c666868c53bf2484f4c696ab8ca0b
0xcec93b333d4140a7d53b6d72ccf4f5f9dff951b6
0x3b0d3ff250ba494db67fee61d7db296e71a73884
0xf57843d682c1fc9c04c11f239812939043e28f0b
0x36acfe929c9697d9d27c6cbf3920339a40a47434
0x0c887611664e9e373ada9736f2e81a732a8f504d
0x8c5b80337379d08011023f7ebcf242929989be99
0xc62d82ac423e46302ca1bbc530dfafb75d14c3ac
0xc83a9eb06f3b2ad4336ce473394e7a1284e76fc1
0x98e8a6c1670ce16072f59d9e96de82d7f92ee9e4
0xd7fb04241cc0af913bd20433cb732ec4bea57c85
0x1889fe8fc8afece4b9094ff1bb8cd478b313e2f3
0xb687343f5505f33b24eaf395d16f0ec8c5268c26
0xba0078dfacc4a06d335b04804e83f4caea095a12
0x563d334175d3999d1db44aab904567ac918db18d
0xb43a4105386cd15a7c8fad7d4d62d5187ab85042
0xdbb24dff1aaad61f5fffe17b85153156d6e567b5
0x50382a8dc7a0a48164a5cff1efe876d1b8efedec
0x12968a90fa56286ca94fc8a851631b7a7e57fc89
0xbe14321450b161610cec206bc2b3c9212ecc2a0f
0x0398cc2a6044d81dea26227a4795d2ed95847253
0xebd8a9c65a5f798670cd488ab957d81e71dbe73d
0xd0ae9c436213ba258cb3fef46733b968c6b3a101
0x54b6bc150a24829f0d93ff96f6a091883903c645
0x1aa18a4c46ec75c5c66593012368469b71049635
0x9817a4866369744ba7130517e676b82e691899ae
0x9e497b978a44f8c3e5f3458090b3d532fe128df1
0x95905e9795c6242e1b70c7dca6cb7ff70afe53fb
0x06d169dae221ace6738894b467b6375b37b13f38
0xc5c22b7afc45ea90e082f4d3b4faea6b30506e8f
0xf4cd502a9eecd2322b453debe9d48a85659670eb
0xeabae352d98de3f3a80633cd4f2d267be101f6cc
0x61e00016d26a700352e8e7ba3868426b000c8e79
0x2d1de4286bd8c78c882a1c2947632fbcc90e55b2
0x2f063960fd4d3581cfa80ff57c5ee882e7725b10
0x5920e62e04d4e5f479e3cdc1b9b90353b9fdb034
0x53ff9b7e00c601dcdfeb9d3da5db4252d72757d6
0x3bf1eb5b1195d8390b8dfbebcd5578a03b6e65d6
0x7c21891e44093d5ebb8f6fd8c4d8b7ee38c64b24
0x50da75dafc419c9f050ea5f5eeec74a23f6812be
0x79bdd2c976685f500a6e0f601b496acff20b56a7
0x60051c88b92ff0bed25e40648d78618f354eb7ad
0xf444d4daee0c2149ae859062e9ae03186bb7c6a3
0x349037d63b6d645369a98af073612c511fffcc45
0xf57cdeb0e7ecbc5be25731721f4400c0bf380443
0xc67415689ff5dcf5f5615a37aaa4e62975ebbaed
0xb1f992e95881445e3ca68b9529750264f341ebc8
0x73f7da0b645000bc05a2ac5c1a157a99ff0c5b58
0x82375d14ff045445e946a7c94fe4e055d83c943d
0x56ff905a89148a61130639083d4de74071d5205a
0x745e2cc8d6a390289522f92c7e0846b4e5e35bd3
0x6ea511a441f60f6838e5bf6c15db6dbe701f2a47
0x1e8b92c75161e6ad8a9fe3ae0259444203b4d528
0x501ac4969cd5a976435911ca203e50fff8f4af1c
0xf8eb0e5f6277504716589300377af653ddb5c72e
0xb4c3a5f129c050960ce9b7b508ac65ab15ee8860
0xf94a4c62902513b721e92e29d9adeebb33a8954f
0x1cbbddd354811fe652b6c9b36506bb171dd7cf6b
0xea8557c92ae738f09ab28ca25307d668f7a0df1c
0x779ced4f86e4ba2de7b303931efee17b0f23430c
0xf264106423e8cd96ce685acad22551a4045cf5ee
0x66dc6977cacf0b182e079c60212e59c35bb16dff
0x2b15f54fb3e97cad56970038c1dc30ec69714549
0x0a4f9b36dec170275c1d33d13607b156b11bddad
0xe1bf344c8be9ede90e3af3badf506dbe0eac79e5
0x49e106e16affd7b3d691246301c4a26cdd7b4b3e
0xf591b6678858044fc55681d0a86227ee13d33a59
0xf94cd5863ddc5687d6b8ba17eaa087685430a1e6
0x54f5d2be819a3d11163a46d57fa1f3176458b1a5
0x1383a030ee82710d3efbe5e580a54919ec0eb73c
0x2377420b05bb2df71a5b130ac0e472a46e3297f0
0x4d15b6860e2059cb62ec564bfde8b963d6a9c4d8
0xd6081dd70267f6e5a3d28ba4cdcc5e7804f703c3
0x37db82ad056447b34aa22feed63fa00fe849c2bf
0x067a12e7741da6ea1fa29d65138338ce5ef0fdda
0x857a92bba11041b79c377f94af358b8b0d895ef5
0x90187b2e3da687e3d48c2ad57928655cdfadac90
0xfc60020375d6eaf86e299870448b88624ca0202f
0xd477304e18c0df37030ee09709b42adb7143eec3
0x413d11063da67333896b4f6a9e01a560dc6ce96c
0x5b575f0c9c1aef1ac6fb5a998d21fbfb8a235f86
0x6c251b5e62a72b6a0ef50b87df5b22d0c8036f29
0xbd795ab17d87410a8806692bf1d37c288a897bbb
0xd2b3ed0730ce8ede88e941cdbadbd86d4f2a5027
0x9f89d33ef6376498465974cedfcb91ecdbdcf44b
0xafae46de8751a4b24d20e6a7b6f6a085350c8a0e
0x787f970f94b891951c90e98509a4b2136786165e
0xece9f0361fd6d970282430075f5b1101304e3572
0xb81850c8fa25e0869c7e392e5ac66c2712a988e4
0xce608793cf8db09bb2e764fbb22779b253870fba
0x06368acb396c5b235f5819aac7eff11f6dea142f
0x8e1e78354c240f28a01da55611d1bbac2f17fe39
0x411ae82edb6877eea6bb7edc16bff7b63c5f097d
0xba584694962da02d59f8616c54c6c91480c27963
0xebf2547ca85fd2d64374cc35c53f3fdaf1f9e9ff
0xe973ed36c5d98302d456e7fd48813b2c6996a2ea
0x3fa861ab8d7dbae2b0eb87aaad2bbd18e4fa4f9c
0x5c07be8ddd9b34bfb993a98f0b9cf044da08156b
0xf473cf81a7b9839e383e08d896a49203dee8fcd5
0x7e704b7fa936e14be948338fc9bec46b192bd101
0xac3232fed5d1a770b8defdf6d2143157fae59bec
0x48a118f444bb4c4bae91b81dd64ae3cce276412a
0x91b14e4b84f83bacdeedf077653beda7f73d25cc
0x005699ff2f1853b7ff9bbba51298e55b583b5ac0
0xe36aaf75975720c0fa3d56087bf910d2a53cc959
0x82734db15b48e2fd68ee39f74b1f8f01acb3fb33
0xef268658776cf63e2d4e732da6cde8f87735de7f
0xfe7435ef05e45017b6abc34152f3ea730dedf304
0x2d0956d1cbb1aff2046e26534b3ffa15e2562785
0x5bf1d97022804c9073e54103d3d298f850fb3794
0x0da5574f8753e759d0f599647e679f598d9e7f9f
0x999dca262b4d353594f3b06e24d55269c4288901
0xd79aac3539b66fb5263042c379479d91e95891cd
0xf038d756f16cf116987054302272fb43766c78ac
0x23670b9ff6f1f4294f222a079fc47f2330e6c88b
0x54e0732671a1a5adb65e45f0aeeed05f1f1c8576
0x7c619ae80041234ba30645f16b3bc9e5acec29cf
0x6405ecae6e64cab2c651e6b99dd9146eb76fe9e5
0x33a930d9d184bfc42929d4b7a734b673bcac3ce0
0x1d52ae89822fc2b8f4561aba13962d33e2c62230
0x336e0bb4b4f8f1e9d38dcf650cb0f096ee3aa09c
0x5698a89ac10842e658804a9c36a8c5085bfbe3c0
0xb775a854bc30d7bcc16b80fe40269ac60781fb14
0xbeabd76e274f9f56c2762e4a5991e3d52f730cfc
0xf8389f241d0240b31c7543ca654c65238830dc2c
0x6e48eb8b629725110d86641253a7b9e1f01b3acd
0xb02a9af0869ecf4c2a20268b4e11c7854563eec9
0xdb251227116046f5076ae327bd2e8b8cffc22479
0x64dbfcbd25d7c99050c8734e7cba52b1276b1a5e
0x16ef625ed2a56074a80d15e2f516dafe7e24fdd0
0x627b707847af94757402404ad88e688d19c44bd0
0x78f79d24e82cf935c622c2d9d9097e8808df4edd
0xd14258621080c38cd87d5141b68b99e346cbafd7
0x027966c8741f4a022106d2f93e5109d9b02127d1
0x5c96a882f6aca7ffb6a42d02a3be9a301f427740
0x379f4da45b98c4ccb95e961cf2acbf0138513095
0x48b7145238bc081e2f4b05b3b3231f55d632e73b
0xe73de476f9d393ca589049dbe64ee3df8413fe4e
0xd64ee2aeac0fdb5f431b79c2aed4a2fcda561d23
0xeb76c07d5d0d5faff3db5907abf3f001d19c5182
0xab5c0f10edd3972b5cd3911c9a81ed2180c218d2
0xe2b72e6d1c56fb97b07e97b835bc62f3057db4f7
0x28d8b65ba344773ef18fb2d833fc602d8433d406
0xdba9f9cfc5d6547478518009c18f898ca174c9f6
0xc47b968c391347a3c614e4210e8d1f73f3d0dd41
0x498922a8a2de93a5f8113caa3ff84be7fa2c6caf
0x1ca6cf7c52cb0747817ce0aca1060b5c47be4180
0x5598e905a263bb1dd86c3e6eb9c4ad9824c63b24
0x408cce46cb8813ffe7a17cbca845be19e23fb9ee
0x3e6910c36939d43ebd8f5283ea2f09c2df00075b
0x00e10186bf1e23e65d58ccd4393e517279068de1
0x1e6b34fcc3d064e709a998625abf5474370a87e0
0x8a85c523345f652c97b025ab93c406bce4761d7c
0x46b7755664089b960238495f8aeedbbb06e7aa7a
0xcce55049d4210bccf57631e5690c4afa58b569bd
0x5455bd87fbf42d1038f738bae93e42feb0e3bcde
0x082d64475a5fc0a76c798de856638931447cdc58
0xedf65b05908c75a447f3b0462c935429011b318e
0x1d6a34214122f6b99a1da73eea9495d18bbaed1a
0xe717ba1093913b47f476f75b0333cfccfbf32ada
0x659d1bf186cbf39a58a557223a3b9276c813be3f
0xf1864b5b6ad0219ff578e25eba1a56c603163c34
0x59fd2e48a8c3b3625a37e975099387afff5b8e53
0x315d7e2ecb515c021db9b2753bacafb94f271b92
0xad4d29208fc527ef87bfe4c6b24c4b571554f92d
0xa63d3836d37146a8b186c7b20258f988b5d74426
0xa243141cf44a1076271d742ed63a2e8fd300eb42
0x15367365cc61c6d3811fa71559bea5ca89030ce3
0xfb097c202b55a850a061c09596b4fc709575e020
0xc2af99b95872603e2d50045901e7692d2d85b95e
0xde1e56dfd2a24ddb0089f4fbb8cf0603d55574ff
0x7af79f67ab00a50b4a08390247cc65b8a9512ab4
0x049a12272e54744c3261a041643f99e40f16f9ef
0x3ea91ad01b48e7318cfde11324594959e8a3e30f
0x38810e93eeb9604cccb2ae5a90f8d1e091e3a73d
0xe36a077e345904b5d95aadc653e94975e8e12467
0xddba2fcaa3a2ce9f5d52bf9d4f228969c6441ef3
0x589214586880a3099517a3fbdfe932939caa65c4
0x36721f87093fd9fc051e0b25e55ea29d26a8fbde
0x88b8de7234376a22a5f23b9a33ae3a97bba564de
0x21f467019584d4226cc1b1958499653b6e4a6c3e
0x0b191f1973c42dd8f1f475752d9272d85f021278
0x980bcbec00b47b87bc61b63193593a355f95391b
0x5de00a5e43b15dd981a6e54c65df8c2d06e9e9aa
0xf7821e1580d63a2f75c330a0f55dbc518c5db36f
0x629db9815745cb540bb6526bb3ba19caf6efe011
0x344c416cc1e616223665c0618d123cada1a948fd
0xa1f4f540ec1a650bbfdb47de51b9f75a61579f0a
0x18db2733f9d2ec520cf6e0dd8f1f267a62fbf544
0x81ac2e5da6e5090f96f022ff359b3a9531261db5
0x7feb7ca6717d47e9211431bb90531527b0b69cae
0x97c63a7d1ad1c0512ff8b5e1ad7ac073de7d0759
0x6157438b82edcb1c1ba7a4332a02e8d00b7ccc20
0x624446485376bb1cd0d784a0ad3aa401569b5f38
0x5c8b77fde512d78097f5ec10db4c20e7d1366225
0xfab3060a346172f16440fd3e85ea63eb8eef92aa
0x5dd2c4b7cee0d63f59fbd40e345dc7fad66550f6
0xde1abd0b45b8845090cee575586d80ae9ed40faa
0xc4d50d21d2bb10243bda8856dfeee1ae97819392
0x08f751b142b227a0416ac6c8f59d194e06b07887

```

# Reasoning

There are a lot of amazing Use cases for a Gnosis Safe but except it's being set up by a **Contract** then it's weird that **one** particular address would set up hundreds  (>=20) of Gnosis safes. 

It's even weirder if they set up these safes between the time the **Safe** token was announced and the cut off date. 

It becomes undoubtedly true that they're probably trying to farm **Safe** tokens if they set up new **wallets** specifically to create the Gnosis safes. 

It gets more weird when they transfer the same token (mostly USDT) to these safe addresses then withdraw it shortly after a few days. We're talking big amounts (Millions). 

My main reasoning was that Safes set up by an address that created multiple safes (>= 20) between the announcement of GIP - 29 & the cutoff date (August 18th), and the address was a newly created wallet with it's first transaction on the same day as they created the Safe and it isn't a **contract / deployer address** then it's definitely an airdrop farmer.

Especially when they're transferring the same token around. 

# Methodology

I used **Dune** to do my research and analysis of the Safes. I also double checked with **etherscan** to exclude safes created by contract deployers and also checking for safes that transferred the same token in and out within a few days span. 

**Dune Methodology**

I firstly selected all the safes that were created between **February 9th 2022** (the date GIP-29 was posted) and **August 18th 2022** (the airdrop cut off date. 

I only looked at the safe set up contract on dune for the latest Gnosis contract **(1.3.0)**.

I joined this results with the **ethereum.transactions** table to find the addresses that initiated the transaction. 

Then I counted the **number of safes** that were created by each address. 

**My first profile was collating all the safes that were created by an address that created more than or equal to **20** safes.** 

I double checked that these addresses were not **contract addresses** using etherscan. There was only one contract address which was **Tokemak Deployer** labelled by etherscan. The rest were EOAs and I excluded the safes created by **Tokemak**. 

I checked all the addresses that deployed a safe and checked the day they made their first ever wallet transaction (Gosh this would have never been possible on Dune V1, thank you Dune). 

I compared this result (using an inner join) with the table that has the date all the safes were created. 

I selected all the **safes** and their creators that were created on the same day that the **creator** made their first ever transaction. 

I then selected all the **Safes** that were created by these creators to include **Safes** that they created on other days. 

**My second profile was joining the two above together**.

In order to identify **safe addresses** that were created to farm the safe airdrop, I used an inner join to compare the two profiles together. 

So basically, a safe address is an airdrop farm address if:

- The safe was created by an address that created multiple (20 or more safes).

**AND**

- The creator of the safe created a Safe on the very same day the creator made their first ever transaction.

Dune dashboard - https://dune.com/Henrystats/safe-airdrop-farmers
Dune Query - https://dune.com/queries/1251338/2154013

I've included a link to etherscan on there dashboard so it's easy to check the wallets individually. 

# Safe Address

0x7C7a1407c35B695E4eE530D80d4bd4C1aF8569E5

**Comment by CaptainTee** [comment]
`So basically, a safe address is an airdrop farm address if:`
`The creator of the safe created a Safe on the very same day the creator made their first ever transaction`

This is a true way of knowing farmers. Well done @henrystats

**Comment by johannesmoormann** [comment]
Many duplicates from older reports but also a few addresses that seem to be relevant.

**Comment by tschubotz** [comment]
Thanks for the report @henrystats . You report contains many Safes that stored significant amounts of USD stable coins for some time. Store of value is legit usage. Your reasoning doesn't explain how you can be sure this would not eliminate legit users.

e.g.
https://gnosis-safe.io/app/eth:0x6de75bb282898ff5e28e308ee7f8750a0f2b077e/transactions/history
https://gnosis-safe.io/app/eth:0x14ab82062dc2622c4f1467042652fb7110b92d53/transactions/history
https://gnosis-safe.io/app/eth:0xefcf76d61503892e184665db37746054670f9966/transactions/history
